testgrid/config.yaml

```diff
@@ -5686,6 +5688,11 @@ dashboards:
     test_group_name: pull-aws-ebs-csi-driver-sanity
     description: aws ebs csi driver sanity test
 
+- name: sig-aws-eks-presubmits
+  dashboard_tab:
+  - name: pull-kubernetes-e2e-1-11-aws-eks-1-11-prod-conformance
+    test_group_name: pull-kubernetes-e2e-1-11-aws-eks-1-11-prod-conformance
```

Comment:
This name can be shorter for better readability on testgrid. For e.g sth like `k8s-1-11-eks-1-11-prod-conformance`?